### ISSUE TITLE
Players and Incentives

### ISSUE THREAD

**Issue by chadwhitacre** [chadwhitacre/openpath#29]
Who are all the participants in the Open Source ecosystem? What are their incentives?

**Comment by andrew** [comment]
What about the users of the software?

**Comment by chadwhitacre** [comment]
lol given enough eyes all bugs are shallow 😅 Updated!

**Comment by andrew** [comment]
Mostly unrelated comment, I find it interesting that the diagram shows all arrows pointing towards maintainers, it doesn't show where other entities are extracting money. It makes it look like everyone is funneling loads of money towards maintainers.

**Comment by chadwhitacre** [comment]
Fair point. It makes better sense in the context of [the talk](https://fosdem.org/2024/schedule/event/fosdem-2024-2751-the-state-of-funding-free-open-source-software/), she's talking specifically about how money gets to maintainers.
